pkg/config/config_template.yaml

```diff
@@ -1083,21 +1084,27 @@ api_key:
   # enabled: "true"
 
   ## @param expvar_port - string - optional - default: 6062
+  ## @env DD_PROCESS_CONFIG_EXPVAR_PORT - string - optional - default: 6062
   ## Port for the debug endpoints for the process Agent.
   #
   # expvar_port: 6062
 
-  ## @param config_port - string - optional - default: 6162
+  ## @param cmd_port - string - optional - default: 6162
```

Comment:
Missing @env line?

Reply:
This one isn't documented in the spreadsheet I'm following, so it is skipped for now.